controllers/admin/AdminModulesController.php

```diff
@@ -1026,7 +1027,7 @@ public function postProcessCallback()
 
             $module_upgraded = implode('|', $module_upgraded);
 
-            if ($key == 'updateAll') {
+            if (Tools::getValue('updateAll')) {
```

Comment:
We are not in the foreach anymore (`foreach ($this->map as $key => $method) {`).
In consequence, testing a value when not in the loop is useless, and brings bugs in some cases (in example, when we do not execute the action updateAll)